config/manifests/bases/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -44,6 +44,14 @@ spec:
         - urn:alm:descriptor:com.tectonic.ui:select:medium
         - urn:alm:descriptor:com.tectonic.ui:select:large
         - urn:alm:descriptor:com.tectonic.ui:select:production
+      - description: The profile controller for IBM Cloud Pak foundational services
```

Comment:
It is an optional parametry for backwardancompatibility, right?

Reply:
Yes, `profileControleller` is optional. It could be empty when creating Common Service CR, and will not cause issue during the upgrade.